src/pages/ContributionsPage/ContributionsPage.tsx

```diff
@@ -142,50 +139,44 @@ const ContributionsPage = () => {
     control,
   });
 
-  const {
-    mutate: createContribution,
-    status: createStatus,
-    reset: resetCreateMutation,
-  } = useMutation(createContributionMutation, {
-    onSuccess: newContribution => {
-      refetchContributions();
-      if (newContribution.insert_contributions_one) {
-        updateSaveStateForContribution(0, 'stable');
-        updateSaveStateForContribution(
-          newContribution.insert_contributions_one.id,
-          'saved'
-        );
-        setCurrentContribution({
-          contribution: {
-            ...newContribution.insert_contributions_one,
-            description: descriptionField.value as string,
-            next: () => data?.contributions[0],
-            prev: () => undefined,
-            idx: 0,
-          },
-          epoch: getCurrentEpoch(data?.epochs ?? []),
-        });
-      } else {
-        updateSaveStateForContribution(0, 'stable');
-        resetCreateMutation();
-      }
-    },
-  });
+  const { mutate: createContribution, reset: resetCreateMutation } =
+    useMutation(createContributionMutation, {
+      onSuccess: newContribution => {
+        refetchContributions();
+        if (newContribution.insert_contributions_one) {
+          updateSaveStateForContribution(0, 'stable');
+          updateSaveStateForContribution(
+            newContribution.insert_contributions_one.id,
+            'saved'
+          );
+          setCurrentContribution({
+            contribution: {
+              ...newContribution.insert_contributions_one,
+              description: descriptionField.value as string,
+              next: () => data?.contributions[0],
+              prev: () => undefined,
+              idx: 0,
+            },
+            epoch: getCurrentEpoch(data?.epochs ?? []),
+          });
+        } else {
+          updateSaveStateForContribution(0, 'stable');
+          resetCreateMutation();
+        }
+      },
+    });
 
   const {
     mutate: mutateContribution,
     status: updateStatus,
     reset: resetUpdateMutation,
   } = useMutation(updateContributionMutation, {
     mutationKey: ['updateContribution', currentContribution?.contribution.id],
-    onSettled: () => {
-      //refetchContributions();
```

Comment:
@topocount I saw this was commented out when this was originally merged and wasn't sure if there was more behind this or not?